backend/src/app/features/components_v2.clj

```diff
@@ -225,8 +225,8 @@
                     [shape]
                     (if (ctk/instance-head? shape)
                       (assoc shape
-                             :type :frame           ; Old groups must be converted
-                             :fills []              ; to frames and conform to spec
+                             :type :frame                 ; Old groups must be converted
+                             :fills (or (:fills shape)[]) ; to frames and conform to spec
                              :hide-in-viewer true
                              :rx 0
                              :ry 0)
```

Comment:
When I wrote this, i forgot that in v1 component roots can be frames, too.

It would be better to put all these five attributes inside a `(cond->` so they are applied only if the type is not already a frame.